frontend/src/container/ReduxTlmDispatcher.jsx

```diff
@@ -74,7 +83,17 @@ export class ReduxTlmDispatcher extends React.Component {
   }
 
   handleContentCreated = data => {
-    this.props.dispatch(addWorkspaceContentList([data.content], data.workspace.workspace_id))
+    if (data.content.content_namespace === 'upload') {
```

Comment:
I think this should be in the reducer.
for workspaceContentList: `id (action.content.content_namespace !== 'content') return`
for workspaceShareFolderContentList: `id (action.content.content_namespace !== 'upload') return`
and for currentWorkspace, I'm sure there is something to handle

Do you think it can be problematic ?

Also, we avoid using hard coded strings like 'upload'. You must add a constant somewhere (util/helper can be a good choice, maybe even frontend_lib) with the values

Reply:
For now, the actions type for content type (content and shareFolderContent) are separated, so the idea is to use only one action type for both type of content ?
Only `[(ADD)(SET)(UPDATE)(DELETE)(RESTORE)]/WORKSPACE_CONTENT` for example

Reply:
That's the idea yes.
I have looked closely into your changes and I don't see any downside for having one action action "contentModfied" and having the reducers doing the intelligence I wrote in my previous comment

Reply:
Because, currently, we have 2 ways of separating them:
- the content.content_namespace
- the action name

While only one is required